services/blockchain-app-registry/shared/metadataIndex.js

```diff
@@ -50,25 +49,27 @@ const getTokensIndex = () => getTableInstance(
 
 const logger = Logger();
 
-const indexTokensInfo = async (filePath, dbTrx) => {
-	const tokensDB = await getTokensIndex();
+const [, , , , repo] = config.gitHub.appRegistryRepo.split('/');
+
+const FILENAME = {
+	APP_JSON: 'app.json',
+	NATIVETOKENS_JSON: 'nativetokens.json',
+};
 
-	const response = await read(filePath);
-	const tokenInfo = JSON.parse(response);
+const KNOWN_CONFIG_FILES = Object.values(FILENAME);
+
+const indexTokensInfo = async (tokenInfo, dbTrx) => {
```

Comment:
Let us please rename `indexTokensInfo` to `indexTokensMetadata`. Similarly, do it for the chain/applications as well.